package.json

```diff
@@ -124,6 +113,12 @@
       "@types/eslint": "8.4.3"
     }
   },
+  "browserslist": [
```

Comment:
Optional: It appears this can be placed in a `.browserlistrc` file as well, if we want to keep this config out of package.json (similar to how we have separate .babbelrc.js files.